### DISCUSSION TITLE
How to get the latest Osm data on your app?

### DISCUSSION THREAD

**Discussion by jdrbaix** [osmandapp/OsmAnd#14027] [Q&A]
Hi,
There were some changes made in my neighbourhood and I wanted to see them in my app. The changes were made 8 days and 22 days ago. Yesterday I updated my map data, but those changes are still not in my app. I do not pay for the app.
What can I do?
Jasper

**Comment by pebogufi** [comment]
this is supported by OsmAnd Live, see
https://osmand.net/features/subscription
or 
https://osmand.net/de/features/subscription
